core/arch/arm/pta/sdp_pa.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <mm/tee_mmu.h>
+#include <sdp_pa.h>
+#include <string.h>
+
+#define PTA_NAME "sdp_pa.pta"
+
+/*
+ * Specificed TA needs to create pagetable item for data
+ * put in SDP memory. Create a PTA for specficed TA to retrieve
+ * SDP memory physical address.
+ */
+static TEE_Result sdp_pa_cmd_virt_to_phys(uint32_t types,
+			     TEE_Param params[TEE_NUM_PARAMS])
```

Comment:
minor: indentation (optee_os sources uses 8 space tabulations)

Reply:
Ok, will fix it in next patch.